<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyJobz</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-utilities">
    <normalon>:/application.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #717980;
}

QPushButton {
	background-color: #c4d4e4 ;
	border-radius: 10px;
 	padding: 7px 10px; 
	border: 1px solid #74A2C9;
}

QPushButton:hover {
    background-color: #a2bad4; 
}

QPushButton:pressed {
    background-color: #7fa0c4; 
}

QPushButton:disabled {
    background-color: #d9d9d9;
}

QCheckBox {
	color: #CED4DA;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout2">
        <item>
         <spacer name="horizontalSpacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="logo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>77</width>
            <height>77</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/resources.qrc">:/application.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>24</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: #CED4DA;
	padding-top: 10px;
}</string>
          </property>
          <property name="text">
           <string>Job Applications</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLineEdit" name="searchLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>666</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget {
    background-color: rgb(230, 230, 230);
	border-radius: 10px;
	padding-left: 8px;
	padding-top: 4px;
	padding-bottom: 4px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hideFinishedCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hide Finished Jobs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="clipboardCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Enable Clipboard Monitoring</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="trayCheckBox">
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Minimize To Tray</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout3">
        <item>
         <widget class="QPushButton" name="addNewItemButton">
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Add New Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="duplicateButton">
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Duplicate Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Delete Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="statsButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Show Statistics</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="font">
         <font>
          <family>Nunito</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
	background-color: #8d959d;
    border: 1px solid #6C757D; 
}

QTableView::item:selected {
	background-color: #6C757D ;
}

QScrollBar:vertical {
    border: 1px solid #8d959d;
    background: #f7f7f7; 
    width: 15px;
    margin: 22px 0 22px 0;
}

QScrollBar::handle:vertical {
    background: #6C757D;
    min-height: 20px;
}

QScrollBar::add-line:vertical {
    background: #C0C0C0;
    height: 20px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    background: #C0C0C0;
    height: 20px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::add-line:vertical {
    border-top: 1px solid #8d959d;
}

QScrollBar::sub-line:vertical {
    border-bottom: 1px solid #8d959d; 
}</string>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>46</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="firstPageButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prevPageButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currPageLabel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1 of 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextPageButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lastPageButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nunito</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar::item:selected {
    background-color: #ADB5BD;
    color: #343A40;
}

QMenu::item:selected {
    background-color: #ADB5BD;
    color: #343A40;
}

QWidget {
    background-color: #495057;
    color: #CED4DA;
}

QMenu::item:disabled {
    background-color: #6c757d;
    color: #adb5bd;
}

QMenu::item:selected:disabled {
    background-color: #6c757d; 
    color: #343a40;
}

QMenuBar::item:disabled {
    background-color: #6c757d;
    color: #adb5bd;
}

QMenuBar::item:selected:disabled {
    background-color: #6c757d; 
    color: #343a40;
}
</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImportFromDB"/>
    <addaction name="actionExportToDB"/>
    <addaction name="actionExportToCSV"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuItemsPerPage">
     <property name="title">
      <string>Items Per Page</string>
     </property>
     <addaction name="separator"/>
     <addaction name="action10Items"/>
     <addaction name="action15Items"/>
     <addaction name="action20Items"/>
     <addaction name="action40Items"/>
    </widget>
    <addaction name="actionEnable_Dark_Mode"/>
    <addaction name="menuItemsPerPage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionGithub"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionEnglish"/>
     <addaction name="actionPortuguese"/>
    </widget>
    <widget class="QMenu" name="menuSetInitialDate">
     <property name="title">
      <string>Set Initial Date</string>
     </property>
     <addaction name="actionToday"/>
     <addaction name="actionBlankDate"/>
    </widget>
    <widget class="QMenu" name="menuSetInitialStatus">
     <property name="title">
      <string>Set Initial Status</string>
     </property>
     <addaction name="actionApplied"/>
     <addaction name="actionBlankStatus"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="menuSetInitialDate"/>
    <addaction name="menuSetInitialStatus"/>
    <addaction name="actionAutoBackup"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEnable_Dark_Mode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enable Dark Mode</string>
   </property>
  </action>
  <action name="actionImportFromDB">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import from Database</string>
   </property>
  </action>
  <action name="actionExportToDB">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Export to Database</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionExportToCSV">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Export to CSV</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionPortuguese">
   <property name="text">
    <string>Portuguese</string>
   </property>
  </action>
  <action name="action10Items">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 Items</string>
   </property>
  </action>
  <action name="action15Items">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>15 Items</string>
   </property>
  </action>
  <action name="action20Items">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>20 Items</string>
   </property>
  </action>
  <action name="action40Items">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>40 Items</string>
   </property>
  </action>
  <action name="actionToday">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Today</string>
   </property>
  </action>
  <action name="actionBlankDate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blank</string>
   </property>
  </action>
  <action name="actionApplied">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Applied</string>
   </property>
  </action>
  <action name="actionBlankStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blank</string>
   </property>
  </action>
  <action name="actionImportFromCSV">
   <property name="text">
    <string>Import from CSV</string>
   </property>
  </action>
  <action name="actionAutoBackup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Auto Backup</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
